packages/rx_bloc_list/pubspec.yaml

```diff
@@ -0,0 +1,37 @@
+name: rx_bloc_list
+description: A new Flutter application.
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
```

Comment:
@DDavidPrime, also you can comment out publish_to

Reply:
Fixed!